app/repositories/CacheRepository.scala

```diff
@@ -151,6 +153,15 @@ class CacheRepository @Inject() (
       totalMatchingDocuments
     )
   }
+
+  def existsLRN(lrn: String): Future[Boolean] = { // TODO: CTCP-3469 set a flag on isSubmitted to potentially to say it is being used elsewhere so we can update it later?. Update README with this new functionality once added.
+    val lrnFilter                    = Filters.eq("lrn", lrn)
+    val notSubmittedFilter           = Filters.eq("isSubmitted", SubmissionState.NotSubmitted.toString)
+    val rejectedPendingChangesFilter = Filters.eq("isSubmitted", SubmissionState.RejectedPendingChanges.toString)
```

Comment:
Minor refactor but could do something like
`val isSubmittedFilter = Filters.in("isSubmitted", SubmissionState.NotSubmitted.toString, SubmissionState.RejectedPendingChanges.toString)`